src/index.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+
+import reducers from './reducers';
+import App from './containers/App';
 import './styles/index.css';
 
+const createStoreWithMiddleware = applyMiddleware()(createStore);
```

Comment:
FYI If you are not using any middleware there is a more concise way of doing this. But happy to leave it in in case you need it later